kubessh/pod.py

```diff
@@ -5,6 +5,7 @@
 import time
 import argparse
 import os
+import sys
```

Comment:
Accidental addition?

Reply:
```suggestion
```

Reply:
This was actually intentional: the code path where the PVC creation fails uses it for exception handling. I'd just missed that before. And yes, we do need better testing ;)